diagnostic.md

```diff
@@ -100,11 +118,18 @@ object.
 ```
 
 ```sh
-# your answer here
+curl --include --request PATCH http://www.example.com/donuts/42
+--data: {"name": "Krüller"}
 ```
 
 Write an AJAX request to change the donut on the server using JSON.
 
 ```js
-let changeDonut = /* your answer here */;
+let changeDonut = function () {
+  return $.ajax({
+    url: 'http://www.example.com' + '/donuts/' + 42,
+    method: 'PATCH',
+    data:'{"name": "French Cruller", "price: "0.99"}'
```

Comment:
We're just renaming the resource.  The data payload should be `{ name: 'Krüller' }`.